guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -633,7 +633,7 @@ been updated.
 Sometimes you will be asked to make some changes to the code you have
 already committed. This can include amending existing commits. In this
 case Git will not allow you to push the changes as the pushed branch
-and local branch do not match. Instead of opening a new pull request,
+and local branch does not match. Instead of opening a new pull request,
```

Comment:
I thought _the pushed branch and local branch_ are plural so you have to use _do_.